#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_SConstruct_debian_mips_el.dpatch by Florian Ernst <florian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Build with lower optimization in order to prevent a compiler error,
## DP: see build logs for 2.40-1 on mips{,el} for details.

@DPATCH@
diff -urNad blender-2.40~/SConstruct blender-2.40/SConstruct
--- blender-2.40~/SConstruct	2006-01-17 14:33:47.000000000 +0100
+++ blender-2.40/SConstruct	2006-01-17 14:34:27.000000000 +0100
@@ -82,7 +82,7 @@
 	build_blender_static = 'false'
 	build_blender_player = 'true'
 	build_blender_plugin = 'true'
-	release_flags = ['-O2']
+	release_flags = ['-O1']
 	debug_flags = ['-O2', '-g']
 	extra_flags = ['-pipe', '-funsigned-char']
 	cxxflags = []
